modules/core/server/routes/core.server.routes.js

```diff
@@ -18,28 +16,9 @@ module.exports = function (app) {
   // Define application route
   app.route('/*').get(core.renderIndex);
 
-//TODO User schema still needs: modulesAssigned 
-
-//TODO Category schema: name, backref:itemList, 
-
   // Render main home for no login, or for public/apply
   app.route('/public/apply').get(core.renderIndex)
   						   .post(core.apply); //For adding user applications
 
-  //TODO enforce login check below this point in controllers.
-  app.route('/items').get(items.list)
-  					.post(items.create);
-  app.route('/items/:itemID').get(items.read)
-  							.put(items.update)
-  							.delete(items.delete);
-  app.param('/itemID', items.itemByID);
-
   //TODO enforce admin login check below this point in controllers.
-  //TODO routing for newuser approval page.
-  app.route('/users').get(users.list)
```

Comment:
These routes for users are pretty necessary, right?

Reply:
No, this route is defunct because we're routing to the controller already from the correct module elsewhere in the codebase. I refactored it so that we use the existing 'admin.server.controller' controller in the 'users' module.